core/custom_filters/custom_filter_wizard_storage.py

```diff
@@ -0,0 +1,118 @@
+                    if cls.__check_object_type(wizard_filter_class, object_type):
+                        cls.__run_load_definition_object_in_wizard(wizard_filter_class, output_of_possible_filters, **kwargs)
+        return output_of_possible_filters
+
+    @classmethod
+    def __run_load_definition_object_in_wizard(
+        cls,
+        wizard_filter_class: CustomFilterWizardInterface,
+        output_of_possible_filters: List[namedtuple],
+        **kwargs
+    ) -> None:
+        """
+        Run the loading of definitions for possible ways of building filters in the provided
+        custom filter wizard class.
+
+        Args:
+            wizard_filter_class (CustomFilterWizardInterface): An instance of the custom filter wizard class
+                                                              responsible for loading filter definitions.
+            output_of_possible_filters (list): A list that contains the definitions of how to build filters.
+                                              Any specific definitions for a particular field will be appended
+                                              to this list.
+
+        Returns:
+            None: This is a void method.
+        """
+        wizard_filter_tuple_type = namedtuple(
+            wizard_filter_class.get_type_of_object(),
+            [cls.__FIELD, cls.__FILTER, cls.__VALUE]
+        )
+        tuple_list_result = wizard_filter_class.load_definition(wizard_filter_tuple_type, **kwargs)
+        output_of_possible_filters.extend(tuple_list_result)
+
+    @classmethod
+    def __create_instance_of_wizard_class(cls, registered_filter_wizard: dict) -> CustomFilterWizardInterface:
+        """
+        Create an instance of the custom filter wizard class.
+
+        Args:
+            registered_filter_wizard (dict): A dictionary containing information about the particular
+                                             registered CustomWizardClass.
+
+        Returns:
+            CustomFilterWizardInterface: An instance of the custom filter wizard class.
+        """
+        return registered_filter_wizard[cls.__KEY_FOR_OBTAINING_CLASS]()
+
+    @classmethod
+    def __check_object_type(cls, wizard_filter_class: CustomFilterWizardInterface, object_type: str) -> bool:
+        """
+        Verify if the object is matched to the object defined in the instance of the class responsible for
+        implementing the wizard interface in the given object type.
+
+        Args:
+            wizard_filter_class (CustomFilterWizardInterface): An instance of the class responsible for
+                                                              implementing the CustomFilterWizardInterface
+                                                              in the given object type.
+            object_type (str): The string representation of the object class that takes part in filtering customization.
+
+        Returns:
+            bool: True if the object is matched, False otherwise.
```

Comment:
```suggestion
```

There's no need for documentation of private functions that do single operation. You could give a single line of explanation why given check is done, but this docstring is excessive.

Reply:
It's the same for __create_instance_of_wizard_class